### DISCUSSION TITLE
ModuleNotFoundError: No module named 'github' but i pip installed pygithub?

### DISCUSSION THREAD

**Discussion by Captain8771** [PyGithub/PyGithub#1964] [Q&A] [answered]
trying to run the file gives: 
```py
================= RESTART: C:/Users/RCG/Downloads/GiTest.py =================
Traceback (most recent call last):
  File "C:/Users/RCG/Downloads/GiTest.py", line 1, in <module>
    from github import Github
ModuleNotFoundError: No module named 'github'
 ```

The code is 
```py
from github import Github

g = Github("token")

for repo in g.get_user().get_repos():
    print(repo.name)
```

Yes i have pip installed pygithub.

**Comment by aahnik** [comment, marked as answer]
Do you have proper installation of python and pip? Many windows users don't have.

It's best to use virtual environments. 

Step 1. Create a virtual environment and activate it ([tutorial](https://youtu.be/APOPm01BVrk) )
Step 2. Run `python --version` and `pip --version` to check if everything all right.
Step 3. Run `pip list` to see list of packages installed
Step 4. Run `pip install --upgrade PyGitHub`
Step 5. To check if the installation succeeded, run `pip show PyGitHub`

**Reply by Captain8771** [reply]
turns out some dependency didnt install, it worked when i did `pip install --upgrade PyGitHub`
Thanks!
